plugins/modules/inventory_workflow_manager.py

```diff
@@ -3144,13 +3508,15 @@ def get_diff_merged(self, config):
                             self.result['response'] = execution_details
 
                             if len(devices_to_add) > 0:
+                                self.device_lists.append(devices_to_add)
```

Comment:
list itself a plural word, give meaningful name.

Reply:
updated the code